extension/src/popup/components/account/AccountList/index.tsx

```diff
@@ -1,22 +1,39 @@
 import React from "react";
 import { Icon } from "@stellar/design-system";
+import { Account } from "@shared/api/types";
 
 import { AccountListIdenticon } from "popup/components/identicons/AccountListIdenticon";
-
-import { Account } from "@shared/api/types";
+import { WalletType } from "@shared/constants/hardwareWallet";
 
 import "./styles.scss";
 
-export const ImportedTag = () => (
-  <span className="AccountList__option-tag">&bull; Imported</span>
-);
+export const OptionTag = ({
+  hardwareWalletType = WalletType.NONE,
+  imported,
+}: {
+  hardwareWalletType?: WalletType;
+  imported: boolean;
+}) => {
+  if (hardwareWalletType) {
+    return (
+      <span className="AccountList__option-tag">
```

Comment:
very NIT: possible to not repeat the element `<span className="AccountList__option-tag">`?

Maybe something like 
```
if (!hardwareWalletType && !imported) return null;
 return <span className="AccountList__option-tag">&bull; {hardwareWalletType || "Imported"} </span>
```

Reply:
ah nice I spent a couple minutes trying to make this a one-ish liner but ended up with a gross "if hw else if imported else null", but I like your way 👍 
https://github.com/stellar/freighter/pull/544/commits/257bb767e7b0886c42d6fa3428855211447280cc